go/vendor/github.com/keybase/go-updater/updater_windows.go

```diff
@@ -22,6 +22,8 @@ func (u *Updater) openApplication(applicationPath string) error {
 func (u *Updater) applyUpdate(localPath string) (err error) {
 	if strings.HasSuffix(localPath, ".exe") {
```

Comment:
"/SILENT" never really did much and won't have an affect on our quieter .msi updates